app/assets/javascripts/modules/get-access-logs.js

```diff
@@ -0,0 +1,73 @@
+moj.Modules.accessLogs = {
+  dropdownName: 'js-access-log-period-dropdown',
+
+  init() {
+    this.$dropdown = $(`#${this.dropdownName}`);
+
+    if (this.$dropdown.length) {
+      this.$form = this.$dropdown.closest('form');
+      this.bindEvents();
+      this.getAccessLogs();
+    }
+  },
+
+  bindEvents() {
+    this.$dropdown.on('change', () => {
+      this.getAccessLogs();
```

Comment:
Can this be rewritten as `this.$dropdown.on('change', this.getAccessLogs);`?

Reply:
Seemingly no – trying that, when `getAccessLogs` runs, the reference to `this` is then referring to the HTML `<select>` element and not the parent module.

Reply:
Ah, I keep getting that wrong.